uui-components/src/buttons/Button.tsx

```diff
@@ -16,20 +41,24 @@ export type ButtonProps = ClickableComponentProps & IDropdownToggler & IHasIcon
     captionCX?: CX;
     /** Icon for drop-down toggler */
     dropdownIcon?: Icon;
+    /** Any HTML attributes (native or 'data-') to put on the underlying component */
+    rawProps?: ButtonRawProps;
 };
 
-export const Button = React.forwardRef<HTMLButtonElement | HTMLAnchorElement | HTMLSpanElement, ButtonProps>((props, ref) => {
+export const Button = React.forwardRef<HTMLButtonElement | HTMLAnchorElement, ButtonProps>((props, ref) => {
     if (__DEV__ && props.captionCX) {
         devLogger.warn('Button: Property \'captionCX\' is deprecated and will be removed in the future release. Please use \'cx\' prop to access caption styles and use cascading to change the styles for the \'uui-caption\' global class');
     }
 
     return (
         <Clickable
             { ...props }
+            href={ props.href }
```

Comment:
why do we pass  href separately, not via {...props}?

Reply:
Typescript can't resolve href using spread